deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -29,21 +30,26 @@
     <param name="table_surface_z"       value="$(arg table_surface_z)"  type="double"/>
 
     <!-- Cloth size params -->
-    <arg name="cloth_x_size"                    value="0.3"/>
+    <arg name="cloth_x_size"                    default="0.3"/>
+    <arg name="cloth_y_size"                    default="0.4"/>
+    <arg name="cloth_num_control_points_x"      default="15"/>
+    <arg name="cloth_num_control_points_y"      default="20"/>
+    <!-- <arg name="cloth_x_size"                    value="0.3"/>
     <arg name="cloth_y_size"                    value="0.5"/>
     <arg name="cloth_num_control_points_x"      value="30"/>
-    <arg name="cloth_num_control_points_y"      value="50"/>
+    <arg name="cloth_num_control_points_y"      value="50"/> -->
 
     <param name="cloth_x_size"                  value="$(arg cloth_x_size)"                 type="double"/>
     <param name="cloth_y_size"                  value="$(arg cloth_y_size)"                 type="double"/>
     <param name="cloth_num_control_points_x"    value="$(arg cloth_num_control_points_x)"   type="int"/>
     <param name="cloth_num_control_points_y"    value="$(arg cloth_num_control_points_y)"   type="int"/>
 
     <!-- Cloth position params -->
-    <arg name="cloth_leading_edge_x"    value="0.06"/>
+    <arg name="cloth_leading_edge_x"    default="0.04"/>
+    <arg name="cloth_leading_edge_y"    default="0.08"/>
 
     <param name="cloth_com_x"           value="$(eval cloth_leading_edge_x + (cloth_x_size / 2.0))" type="double"/>
-    <param name="cloth_com_y"           value="0.0"                                                 type="double"/>
+    <param name="cloth_com_y"           value="$(eval cloth_leading_edge_y + 0.0)"                  type="double"/>
```

Comment:
This value makes no sense. The center of mass of the cloth cannot be at the same place as leading edge of the cloth.

Reply:
We still need to do something about this.